### PR TITLE
feat(conf): custom html file

### PR DESCRIPTION
**Pull request by feugy**
Allows users to specify a different html file with configuration

### CODE REVIEW

**Comment by feugy** [comment]
Here we are: a very simplistic change that will allow users to specify their own Html file.

File: lib/index.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function makeRouteHandler(options, userContent) {
   const WEBPACK_DEV = options.webpackDev;
   const RENDER_JS = options.renderJS;
   const RENDER_SS = options.serverSideRendering;
-  const html = fs.readFileSync(Path.join(__dirname, "index.html")).toString();
+  const html = fs.readFileSync(options.htmlFile).toString();
```
Comment (ananavati) [inline lib/index.js]: like the idea of passing a custom HTML but by default it should always load Path.join(__dirname, "index.html") and if `options.customHtml` passed `customHtml` path would be used.

**Comment by jchip** [comment]
thanks for the PR.

**Comment by feugy** [comment]
You're welcome !
Thanks a lot gents.

**Pull request merged**